docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -42,48 +68,15 @@ To create the unit test project, do the following:
 
    - It applies the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute to define `TestMethod1` as a test method for automatic execution when the unit test is run.
```

Comment:
"...Attribute attribute" sounds awkward -- this could be changed to "...the [TestMethod] attribute..."